### PR TITLE
Chore: Update README.md

### PR DESCRIPTION
**Pull request by JamesHenry**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,25 +40,7 @@ estraverse.traverse(ast, {
...
-
-And there are tools constructed on Escope.
-
-- [Esmangle](https://github.com/estools/esmangle) is a minifier / mangler / optimizer.
-- [Eslevels](https://github.com/mazurov/eslevels) is a scope levels analyzer and [SublimeText plugin for scope context coloring](https://github.com/mazurov/sublime-levels) is constructed on it.
-- [Esgoggles](https://github.com/keeyipchan/esgoggles) is JavaScript code browser.
-
-
-### License
+### Original Escope License
```
Comment (nzakas) [inline README.md]: We don't actually need this, as the original license is in the LICENSE file. 

Also, we need a section on contributing, build commands, and current license. You can copy those from the TypeScript parser readme.

**Comment by JamesHenry** [comment]
Updated, PTAL.

(I also made the lint task run as part of the test task to align it with typescript-eslint-parser description)

File: README.md [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
-[![Build Status](https://travis-ci.org/estools/escope.png?branch=master)](https://travis-ci.org/estools/escope)
+ESLint Scope is the [ECMAScript](http://www.ecma-international.org/publications/standards/Ecma-262.htm) scope analyzer used in ESLint. It is a fork of [escope](http://github.com/estools/escope).
 
-### Example
+## Usage
+
+Install:
+
+```
+npm i eslint-scope --save-dev
```
Comment (nzakas) [inline README.md]: Oops, probably just `--save` here?

**Comment by JamesHenry** [comment]
*sigh* Typing is hard 😄

**Pull request merged**